src/types/config.ts

```diff
@@ -7,8 +7,8 @@ export interface ILintConfig {
   encoding: string;
   format: OutputFormat;
   output?: string;
-  ruleset?: string[];
-  skipRule?: string[];
+  ruleset?: string | string[];
+  skipRule?: string | string[];
```

Comment:
Would it be worth to make this always an array so you do not have to deal with this?